tests/monitoring/component_monitoring.go

```diff
@@ -245,6 +234,8 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 				_, _ = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(context.Background(), vmi)
 				_ = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Delete(context.Background(), vmi.Name, &metav1.DeleteOptions{})
 
+				g.Expect(libmonitoring.CheckAlertExists(virtClient, virtController.restErrorsBurtsAlert)).To(BeTrue())
+				g.Expect(libmonitoring.CheckAlertExists(virtClient, virtController.restErrorsHighAlert)).To(BeTrue())
```

Comment:
Perhaps it will be better to check this after deleting the virt-controller CRB and not after both are deleted.